Blizzard.IO.RabbitMQ.Builders/Rpc/RpcConnectionKey.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using System;
+
+namespace Blizzard.IO.RabbitMQ.Builders.Rpc
+{
+    public struct RpcConnectionKey
```

Comment:
It's not really a `key` but more of an `id`, these five parameters are defining the identity of a connection, consider rename `RpcConnectionId`

Reply:
f